scripts/cleanup_old_operations_table.py

```
@@ -0,0 +1,300 @@
+            new_table = self.dynamodb.Table(self.new_table_name)
+            
+            old_response = old_table.describe_table()
+            new_response = new_table.describe_table()
+            
+            old_count = old_response['Table']['ItemCount']
+            new_count = new_response['Table']['ItemCount']
+            
+            logger.info(f"Old table item count: {old_count}")
+            logger.info(f"New table item count: {new_count}")
+            
+            return old_count, new_count
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)